app/Repositories/BusinessTypeRepository.php

```diff
@@ -12,4 +12,16 @@ public function model()
     {
         return BusinessType::class;
     }
+    /** 
+     * @var string $businessTypeName "pay" || "cash"
+     * 
+     **/
+    public function getBusinessTypesId(string $businessTypeName = 'pay')
```

Comment:
what does this function do?

Reply:
it returns array of ids from business_types table based on the string passed.

Accepts cash or pay.

if cash is passed in, it will return all ids of business type that is of type Altaray Pay.

I created it why trying to debug yesterday, left it, felt it might come useful someday.